README.md

```diff
@@ -98,30 +109,44 @@ for more information.
 
 To finish setting up Chains, please complete the following steps:
 
-* [Add authentication to the Chains controller](docs/authentication.md)
-* [Generate a cryptographic key and configure Chains to use it for signing](docs/signing.md)
-* [Set up any additional configuration](docs/config.md)
+- [Add authentication to the Chains controller](docs/authentication.md)
+- [Generate a cryptographic key and configure Chains to use it for signing](docs/signing.md)
+- [Set up any additional configuration](docs/config.md)
 
 ## Tutorials
 
-To get started with Chains, try out our [getting started tutorial](docs/tutorials/getting-started-tutorial.md).
+To get started with Chains, try out our
+[getting started tutorial](docs/tutorials/getting-started-tutorial.md).
 
-To start signing OCI images and generating signed provenance for them, try our [signed provenance tutorial](docs/tutorials/signed-provenance-tutorial.md).
+To start signing OCI images and generating signed provenance for them, try our
+[signed provenance tutorial](docs/tutorials/signed-provenance-tutorial.md).
+
+### Community tutorials
+
+The Chains community has been hard at work is sharing some resources as well :
+
+- [Dual storage backend setup](https://github.com/thesecuresoftwarefactory/ssf/blob/main/examples/buildpacks/dual-storage-backend.md) showcases how to use multiple storage backends and
+verify the attestations with [cosign].
```

Comment:
nit, missing the link to cosign

Reply:
It is at the very bottom of the file.

Reply:
i see it! but it still doesn't seem to be resolving 🤔 

![image](https://user-images.githubusercontent.com/10676490/155656779-3b235741-4b5e-44ab-895e-cf16af7cdcf1.png)

Reply:
Weird, I see it as resolved though:

<img width="731" alt="image" src="https://user-images.githubusercontent.com/6969134/155730925-2b6c817f-70e1-4141-b7f7-d93aeee874cb.png">

Reply:
let's merge and see 😄